### DISCUSSION TITLE
Location reminders

### DISCUSSION THREAD

**Discussion by dovry** [runelite/runelite#15502] [Ideas]
You set a reminder to notify you when you're in the specified area
When you cross into chunks/areas a notification with your text is shown
My usecase would be for running clues/multiple quests at the same time to not forget certain items or to do my clues while I'm in the area
You could also create a reminder to buy more supplies when you're close to the G.E.

No alternatives exist currently that I know of, aside from keeping notes and checking them any time you're about to do something

**Additional context**
Concept on [iOS](https://www.macrumors.com/how-to/create-location-based-reminder-ios/)
Concept on [Android](https://9to5google.com/2019/11/04/set-up-location-reminders-google-keep/)

**Comment by Nightfirecat** [comment]
You can place ground markers and add labels to them as reminders to do those activities

**Comment by dovry** [comment]
> You can place ground markers and add labels to them as reminders to do those activities

that’s a good suggestion, but if you’re walking around, let’s say Asgarnia, and get a clue in Ardougne. You could enter from 10 different areas or teleports and not see the tile for days
